zh/deploy-on-gcp-gke.md

```diff
@@ -8,728 +8,346 @@ aliases: ['/docs-cn/tidb-in-kubernetes/dev/deploy-on-gcp-gke/']
+NAME                     TYPE           CLUSTER-IP      EXTERNAL-IP      PORT(S)               AGE
+basic-grafana            LoadBalancer   10.15.255.169   34.123.168.114   3000:30657/TCP        35m
 ```
 
-### 自定义日志
+其中 `EXTERNAL-IP` 栏即为 LoadBalancer IP。
 
-GKE 使用 [Fluentd](https://www.fluentd.org/) 作为其默认的日志收集工具，然后将日志转发到 Stackdriver。Fluentd 进程可能会占用大量资源，消耗大量的 CPU 和 RAM。[Fluent Bit](https://fluentbit.io/) 是一种性能更高，资源占用更少的替代方案。与 Fluentd 相比，更建议在生产环境中使用 Fluent Bit。可参考[在 GKE 集群中设置 Fluent Bit 的示例](https://github.com/pingcap/k8s-fluent-bit-stackdriver)。
+你可以通过浏览器访问 `<grafana-lb>:3000` 地址查看 Grafana 监控指标。其中 `<grafana-lb>` 替换成前面获取的 IP。
 
-### 自定义节点池
+Grafana 默认登录信息：
 
-集群是按区域 (regional) 而非按可用区 (zonal) 来创建的。也就是说，GKE 向每个可用区复制相同的节点池，以实现更高的可用性。但对于 Grafana 这样的监控服务来说，通常没有必要维护相同的可用性。你可以通过 `gcloud` 手动删除节点。
+- 用户名：admin
+- 密码：admin
 
-> **注意：**
->
-> GKE 节点池通过实例组管理。如果你使用 `gcloud compute instances delete` 命令删除某个节点，GKE 会自动重新创建节点并将其添加到集群。
+## 升级 TiDB 集群
 
-如果你需要从监控节点池中删掉一个节点，可采用如下步骤：
+要升级 TiDB 集群，可以通过 `kubectl edit tc basic -n tidb-cluster` 修改 `spec.version`。
 
-1. 获取托管的实例组和所在可用区。
+升级过程会持续一段时间，你可以通过 `kubectl get pods -n tidb-cluster --watch` 命令持续观察升级进度。
 
-    {{< copyable "shell-regular" >}}
+## 扩容 TiDB 集群
 
-    ```bash
-    gcloud compute instance-groups managed list | grep monitor
-    ```
+注意扩容前需要对相应的节点池进行扩容，以便新的实例有足够的资源运行。
 
-    输出结果类似：
-
-    ```
-    gke-tidb-monitor-pool-08578e18-grp  us-west1-b  zone   gke-tidb-monitor-pool-08578e18  0     0            gke-tidb-monitor-pool-08578e18  no
-    gke-tidb-monitor-pool-7e31100f-grp  us-west1-c  zone   gke-tidb-monitor-pool-7e31100f  1     1            gke-tidb-monitor-pool-7e31100f  no
-    gke-tidb-monitor-pool-78a961e5-grp  us-west1-a  zone   gke-tidb-monitor-pool-78a961e5  1     1            gke-tidb-monitor-pool-78a961e5  no
-    ```
+下面是将 GKE 集群 `tidb` 的 `tikv` 节点池扩容到 6 节点的示例：
 
-    第一列是托管的实例组，第二列是所在的可用区。
-
-2. 获取实例组中的实例名字。
-
-    {{< copyable "shell-regular" >}}
-
-    ```bash
-    gcloud compute instance-groups managed list-instances ${instance_group} --zone ${zone}
-    ```
+{{< copyable "shell-regular" >}}
 
-    示例：
+```shell
+gcloud container clusters resize tidb --node-pool tikv --num-nodes 2
+```
 
-    {{< copyable "shell-regular" >}}
+> **注意：**
+> 
+> 在区域集群下，节点分别创建在 3 个可用区下。这里扩容后，节点数为 2 * 3 = 6 个。
 
-    ```bash
-    gcloud compute instance-groups managed list-instances gke-tidb-monitor-pool-08578e18-grp --zone us-west1-b
-    ```
+然后通过 `kubectl edit tc basic -n tidb-cluster` 修改各组件的 `replicas` 为期望的新副本数进行扩容。
 
-    输出结果类似：
+更多节点池管理可参考 [node pools 文档](https://cloud.google.com/kubernetes-engine/docs/concepts/node-pools)。
 
-    ```
-    NAME                                       ZONE        STATUS   ACTION  INSTANCE_TEMPLATE                     VERSION_NAME  LAST_ERROR
-    gke-tidb-monitor-pool-08578e18-c7vd  us-west1-b  RUNNING  NONE    gke-tidb-monitor-pool-08578e18
-    ```
+## 部署 TiFlash/TiCDC
 
-3. 通过指定托管的实例组和实例的名称来删掉该实例。
+### 新增节点组
 
-    例如：
+为 TiFlash/TiCDC 各自新增一个节点组。
```

Comment:
Please provide two separate commands for TiFlash and TiCDC.

Reply:
done